packages/react-native-payments/src/components/CreditCard.js

```diff
@@ -0,0 +1,86 @@
+import { OPERATION_SOURCE, OPERATION_TYPE } from './constants';
+import { useAuthentication } from '@farfetch/blackout-react/authentication/hooks';
+import { WebView } from 'react-native-webview';
+import React, { useCallback, useMemo } from 'react';
+
+const ADD_LISTENER_JS =
+  "window.addEventListener('message', listener => { window.ReactNativeWebView.postMessage(JSON.stringify(listener.data)); });";
+
+const CreditCard = ({
+  additionalProperies,
+  creditCardDispatch,
+  paymentIntentId,
+  webViewRef,
+}) => {
+  const {
+    activeTokenData: {
+      data: { accessToken },
+    },
+  } = useAuthentication();
+
+  const webViewSource = useMemo(
+    () => ({
+      uri: `${additionalProperies?.url}?paymentIntentId=${paymentIntentId}&staticName=${additionalProperies?.staticName}&folderName=${additionalProperies?.folderName}&locale=${additionalProperies?.locale}`,
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+        Referer: 'http://dev.blackandwhite-ff.com', // TODO - For testing purpuses only
+      },
+    }),
+    [accessToken, additionalProperies, paymentIntentId],
+  );
+
+  const updateStateWithOperationResult = useCallback(
+    event => {
+      const data = JSON.parse(event?.nativeEvent?.data);
+      const source = data?.source;
+
+      if (source !== OPERATION_SOURCE) {
+        return;
+      }
+
+      const operation = data?.operation;
+
+      if (operation === OPERATION_TYPE.VALIDATION) {
+        creditCardDispatch({ isFormValid: data?.isFormValid });
+      }
+
+      const payload = data?.payload;
+
```

Comment:
You don't need to leave empty lines on every `const`, you can stick them together 🙂 

Please review this everywhere 🙏

Reply:
Should be ok now, please double check 👍

Reply:
Hmm, it still seems that there are unnecessary spaces (lines 46 and 48)

Reply:
Sorry about that, missed it. Should be ok now 👍